docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -134,15 +134,18 @@ The ability to download recordings for offline viewing will be available in a fu
 
 ### Will Teleport be automatically upgraded with each release?
 
-Teleport Cloud follows the Teleport [Production
-readiness](../../upcoming-releases.mdx) guidelines.
+Teleport Cloud follows the Teleport [Upcoming
+Releases](../../upcoming-releases.mdx) timelines.
 
 - Major version upgrades for customer Auth and Proxy Servers occur after the
   first minor release (for example (=teleport.major_version=).1.0) of Teleport.
   This typically happens within one month of a new major release of Teleport
   becoming available.
 - After the first minor release, subsequent minor and patch upgrades typically
   occur within a week.
+- Your Teleport tenant will NOT be automatically upgraded to the next major version if any
+  Teleport agents connected to the cluster are not compatible (> 1 major version behind).
+  Once corrected, Teleport will upgrade at the next scheduled maintenance window.
```

Comment:
```suggestion
```

This last part is not necessarily true, we're working on a better process for major version upgrades that would perform this check everytime.  Right now a list of customers that are ready to upgrade is compiled at an non-scheduled cadence though over time customers who are ready will get upgraded (eventually consistent).